Programs/_testembed.c

```diff
@@ -1603,6 +1648,7 @@ struct TestCase
 static struct TestCase TestCases[] = {
     {"test_forced_io_encoding", test_forced_io_encoding},
     {"test_repeated_init_and_subinterpreters", test_repeated_init_and_subinterpreters},
+    {"test_finalize_subinterps", test_finalize_subinterps},
```

Comment:
There needs to be a matching test case in `Lib/test/test_embed.py`.

Reply:
You're right, I missed that!